server/graphql/v2/mutation/RecurringExpenseMutations.ts

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+import { GraphQLBoolean, GraphQLNonNull } from 'graphql';
+
+import expenseStatus from '../../../constants/expense_status';
+import models from '../../../models';
+import { NotFound, Unauthorized } from '../../errors';
+import { idDecode, IDENTIFIER_TYPES } from '../identifiers';
+import { RecurringExpenseReferenceInput } from '../input/RecurringExpenseReferenceInput';
+
+const recurringExpenseMutations = {
+  cancelRecurringExpense: {
+    verifyExpense: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      description: 'To verify and unverified expense.',
+      args: {
+        recurringExpense: {
+          type: new GraphQLNonNull(RecurringExpenseReferenceInput),
+          description: 'Reference of the expense to process',
+        },
+      },
+      async resolve(_: void, args, req: express.Request): Promise<boolean> {
+        if (!req.remoteUser) {
+          throw new Unauthorized();
+        }
+
+        const id = idDecode(args.recurringExpense.id, IDENTIFIER_TYPES.RECURRING_EXPENSE);
+        const recurringExpense = await models.RecurringExpense.findByPk(id);
+        if (!recurringExpense) {
+          throw new NotFound();
+        }
+        if (![recurringExpense.CollectiveId, recurringExpense.FromCollectiveId].some(req.remoteUser.isAdmin)) {
```

Comment:
We need to use `isAdminOfCollective` in case the expense is submitted from/to an event or project

Reply:
To be honest, I'm not even using this mutation because deleting the draft itself will delete the recurring expense.
Should I just remove?